app/Ptr/PtrUpdateService.php

```diff
@@ -44,6 +51,11 @@ protected function updateAll(Collection $items) {
   }
 
   private function setPtr(Collection $items) {
+    foreach($items as $ptr){
+      if(!$this->ptrValidator->validate($ptr->ip, $this->input('ptr'))){
+        abort(409, "Invalid IP");
```

Comment:
please provide a more descriptive explanation of what's invalid about the IP. This needs to be easily actionable by the user. If you were a user seeing this, would you know what you needed to do to resolve it?